extensions/amp-font/0.1/amp-font.js

```diff
@@ -39,25 +39,18 @@ import {Services} from '../../../src/services';
 import {isFiniteNumber} from '../../../src/types';
 import {user} from '../../../src/log';
 
```

Comment:
Is this unnecessary because they are private and not exported?

Reply:
Yep, and they are assigned primitives so Closure can easily infer the type automatically. @erwinmombay please correct me if I am mistaken